x-pack/plugins/infra/public/pages/metrics/inventory_view/components/bottom_drawer.tsx

```diff
@@ -67,12 +67,13 @@ export const BottomDrawer: React.FC<{
 
 const BottomActionContainer = euiStyled.div<{ isOpen: boolean }>`
   padding: ${(props) => props.theme.eui.paddingSizes.m} 0;
-  position: fixed;
+  position: absolute;
+  height: ${(props) => (props.isOpen ? '244px' : '48px')};
+  overflow: ${(props) => (props.isOpen ? 'visible' : 'hidden')};
   left: 0;
   bottom: 0;
   right: 0;
-  transition: transform ${TRANSITION_MS}ms;
-  transform: translateY(${(props) => (props.isOpen ? 0 : '224px')})
+  transition: height ${TRANSITION_MS}ms;
```

Comment:
`height` transitions aren't hardware-accelerated, so this significantly impacts performance of the open/close animation.

Is switching the transition to `height` necessary? In my quick test, just `position: absolute` fixes this issue in Chrome and Firefox, unless I'm missing something?

Reply:
Unfortunately, if you try to scroll down once you get to the bottom you will see that the timeline area is being displayed when you `position: absolute` due to it no longer being positioned relative to the viewport.  The `overflow: hidden` property is needed to actually hide it, but that makes having the same transition effect on transform not possible (as far as I could tell).

Reply:
@Zacqary Unfortunately was not able to find a workaround to have the same animation effect using translateY instead of height in this scenario.